src/main/java/net/sf/jabref/exporter/FieldFormatterCleanups.java

```diff
@@ -85,7 +85,7 @@ public int hashCode() {
         return Objects.hash(actions, enabled);
     }
 
-    private static List<FieldFormatterCleanup> parse(String formatterString) {
+    public static List<FieldFormatterCleanup> parse(String formatterString) {
```

Comment:
Using the constructor https://github.com/obraliar/jabref/blob/c948c623fa212b29169f31f64b848fc1bf3bb9e8/src/main/java/net/sf/jabref/exporter/FieldFormatterCleanups.java#L44 does not work?